Article/Article.md

```diff
@@ -120,6 +144,10 @@ The second check has been the introduction of a different gender equality indica
 
 - Lastly, the most important figure to look at is the variable conditioning for the whole analysis, because the gender index is very often intertwined to it, and because in general one can immediately get an answer to the question "how much variance is explained by the variable chosen, independently by the country economic development". The result of this analysis is that the gender differences residualised on the economic develooment do not show a dependency on the GDI (also residualised on economic development), while the vice-versa applies (meaning, there is a strong and statistically significant correlation between gender differences and economic development, both residualised on the GDI). This result arises many questions that are beyond the scope of this analysis.
 
+[the role of coveing different levels of power]
+
+
+Within its design, WEF index shows that no coutry in the world are approaching equality in econtomic and political domain, while the gap in education and health is almost closed [REF].
```

Comment:
Shouldn't this be more into the introduction? It doesn't sound like something derived from our analysis